src/style.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+body {
+  margin: 0;
+  --content-text-color: #060b35;
+  --button-color: #00acdc;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container {
+  display: flex;
+  padding: 0 3px;
+  margin-bottom: 40px;
+}
+
+.card__imac-img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--content-text-color);
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {

```

Comment:
Move these styles to a separate file 
![image](https://user-images.githubusercontent.com/96409563/219471753-6eeaf14d-b0a5-4cf6-9390-0be74afbdf91.png)